src/main/java/BobChat.java

```diff
@@ -0,0 +1,135 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import Errors.*;
+
+class Bob {
+    private final Scanner scanner = new Scanner(System.in);
+    private final ArrayList<Task> tasks = new ArrayList<>();
+
+    public void start() throws InputExceptions {
+        System.out.println("____________________________________________________________");
+        ASCII_Art.printArt();
+        System.out.println(" Hello! I'm BobChungus ");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+
+        while (true) {
+            String input = scanner.nextLine().trim();
+            String[] inputParts = input.split(" ", 2);
+            String command = inputParts[0].toLowerCase();
+            String arguments = (inputParts.length > 1) ? inputParts[1] : "";
+
+            try {
+                switch (command) {
+                    case "bye":
+                        System.out.println("____________________________________________________________");
```

Comment:
Since this line of code is repeating at multiple instances, you might want to save it in a variable or make it a function which you can reference.